CHANGELOG.md

```diff
@@ -27,6 +27,32 @@ All notable changes to this project will be documented in this file.
 * ### Major Changes
     * The signature of the collection classes' __init__ methods have been changed and any code that constructs these objects directly should be updated to get them from the appropriate object (e.g. System.devices, Task.channels)
 
+## 0.8.0
+
+* ### Merged Pull Requests
+    * [Query: Closed PRs with the label: interpreter_implementation](https://github.com/ni/nidaqmx-python/issues?q=label%3Ainterpreter_implementation+is%3Aclosed)
+    * [Query: Closed PRs with the label: library_interpreter](https://github.com/ni/nidaqmx-python/issues?q=label%3Alibrary_interpreter+is%3Aclosed)
+    * [Query: Closed PRs with the label: grpc_interpreter](https://github.com/ni/nidaqmx-python/issues?q=label%3Alibrary_interpreter+is%3Aclosed)
+    * [Query: Closed PRs with the label: test_improvements to increase the test coverage](https://github.com/ni/nidaqmx-python/issues?q=label%3Atest_improvements+is%3Aclosed)
+    * [Query: Closed PRs with the label: interpreter_fixes to fix all the issues in the interpreters](https://github.com/ni/nidaqmx-python/issues?q=label%3Ainterpreter_fixes+is%3Aclosed)
+    * [Query: Closed PRs with the label: interpreter_testcase_update to make the existing test cases run with both the interpreter](https://github.com/ni/nidaqmx-python/issues?q=label%3Ainterpreter_testcase_updates+is%3Aclosed)
+    * [Query: Closed PRs with the label: event_handling that takes care of the event handling in the interpreters](https://github.com/ni/nidaqmx-python/issues?q=label%3Aevent_handling+is%3Aclosed)
+* ### Resolved Issues
+    * ...
+* ### Major Changes
+    * Added support to communicate with the DAQmx devices through gRPC.
+    * For communicating with the device through gRPC, the following changes have been made:
+        * Added a stub generator which will generate the gRPC stub files based on the proto files present in `src/codegen/protos`
+        and the files will be generate into `generator/nidaqmx/_stubs`.
+        * Updated the existing generator to generate the `grpc_interpreter` based on the metadata provided, this will implement all the necessary methods using the gRPC stubs.
+        * Created `grpc_session_options` which defines the necessary information that is required to communicate via gRPC.
+    * Updated task, system and scale objects to taken in `grpc_session_options` as an optional parameter and based on this information, the interpreter to be used will be decided.
+    * Updated the existing tests to run with and without gRPC support.
+    * Added multiple new test cases to improve the test coverage.
+    * Currently, the behavior of all the operations will be similar when communicating with and without gRPC with the following exceptions:
+        * When reading values from a `read_stream`, the initial values sent by the user will not retained in the read array. Based on the number of samples read, the unread samples will be set to default value of the datatype specified.
```

Comment:
This belongs in a "known issues" list. 

Are there any other known issues? (excluding tech debt)